### ISSUE TITLE
Uncaught TypeError: Cannot read property 'destroy' of undefined

### ISSUE THREAD

**Issue by MavisCletis** [morassman/atom-commander#84]
[Enter steps to reproduce:]

1. ...
2. ...

**Atom**: 1.19.7 x64
**Electron**: 1.6.9
**OS**: Microsoft Windows 10 Pro
**Thrown From**: [atom-commander](https://github.com/morassman/atom-commander) package 0.10.2

### Stack Trace

Uncaught TypeError: Cannot read property 'destroy' of undefined

```
At C:\Users\jselleck\.atom\packages\atom-commander\lib\views\diff\diff-view.coffee:32

TypeError: Cannot read property 'destroy' of undefined
    at DiffView.module.exports.DiffView.initialize (/packages/atom-commander/lib/views/diff/diff-view.coffee:32:87)
    at /packages/atom-commander/node_modules/space-pen/lib/space-pen.js:184:25)
    at /packages/atom-commander/lib/views/diff/diff-view.coffee:11:5)
    at Function.module.exports.Utils.compareFiles (/packages/atom-commander/lib/utils.coffee:15:16)
    at Actions.module.exports.Actions.compareFiles (/packages/atom-commander/lib/actions.coffee:258:11)
    at MenuItem.callback (/packages/atom-commander/lib/actions.coffee:1:1)
    at MenuBarView.module.exports.MenuBarView.handleMenuItem (/packages/atom-commander/lib/views/menu/menu-bar-view.coffee:135:18)
    at MenuBarView.module.exports.MenuBarView.handleKeyUp (/packages/atom-commander/lib/views/menu/menu-bar-view.coffee:130:8)
    at AtomCommanderView.module.exports.AtomCommanderView.handleKeyUp (/packages/atom-commander/lib/atom-commander-view.coffee:151:16)
    at /packages/atom-commander/lib/atom-commander-view.coffee:110:20)
    at HTMLDivElement.dispatch (/packages/atom-commander/node_modules/jquery/dist/jquery.js:4435:9)
    at HTMLDivElement.elemData.handle (/packages/atom-commander/node_modules/jquery/dist/jquery.js:4121:28)
```

### Commands

```
     -4:31.9.0 core:backspace (input.hidden-input)
  3x -4:30.8.0 core:confirm (input.hidden-input)
 10x -3:29.5.0 core:backspace (input.hidden-input)
     -3:26.1.0 core:confirm (input.hidden-input)
     -1:02.1.0 core:backspace (input.hidden-input)
     -0:52.5.0 core:confirm (input.hidden-input)
  4x -0:48.9.0 core:backspace (input.hidden-input)
     -0:42.8.0 core:confirm (input.hidden-input)
```

### Non-Core Packages

```
atom-beautify 0.30.5 
atom-commander 0.10.2 
expand-selection-to-quotes 0.11.0 
highlight-selected 0.13.1 
language-vb 0.1.0-pre1 
```

**Comment by MavisCletis** [comment]
This happens every time I try to compare two files with atom commander.

**Comment by morassman** [comment]
Hi @MavisCletis. Thanks for reporting this. 

I've fixed the bug. It will be part of the next update.

**Comment by joshupetersen** [comment]
If you don't mind me asking, when's the next update coming?

**Comment by morassman** [comment]
Hi @joshupetersen. You're welcome to ask

I intended to address other issues before making the next release, but just haven't been able to get round to them. I've recently changed jobs and have been rather swamped, but I'm not trying to make excuses :)

Instead of trying to get other things into the next release as well I'll just make a smaller update with this bug fix in it. I just want to test a bit before then, but I'll get it out by the weekend.
